### ISSUE TITLE
Quickstacked items do not restock items in the second half of a chest's inventory

### ISSUE THREAD

**Issue by Lucipo** [Tfarcenim/DropOff#24]
MC Version: 1.19.2
Mod Version: 1.19.2-6

Issue: Items that are quickstacked into a chest with said items in the latter 27 slots of a chest automatically appear in the top half of items within the chest, as seen below
![image](https://user-images.githubusercontent.com/36358294/216735345-b27a96d8-5793-422d-8433-6f07637cdd64.png)
![image](https://user-images.githubusercontent.com/36358294/216735356-92a20edc-5a3d-4345-be6e-b5be02abe744.png)
![image](https://user-images.githubusercontent.com/36358294/216735367-3c4e88d0-37ab-4c19-aeb1-30f8d85b7677.png)

Basically makes quickstacking into chests that are manually sorted useless because anything in the 2nd chest appears in the top part

**Comment by Lucipo** [comment]
Also preferred that a patch is issued on a 1.19.2 version, if alongside a 1.19.3 version

**Comment by Tfarcenim** [comment]
fixed in6.1
